repo/packages/E/elastic/900/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "packagingVersion": "4.0",
+  "upgradesFrom": [
+    "*"
```

Comment:
+1 to @mpereira not sure why this is `*`. Should be version specific.

Reply:
Raised new PR with version specific https://github.com/mesosphere/universe/pull/2447